test-dxcompiler-in-project/replace_dxcompiler_jar.sh

```diff
@@ -0,0 +1,11 @@
+#/bin/bash
```

Comment:
This script was removed in https://github.com/dnanexus/dxCompiler/pull/228/files. Why reintroduce it?

Reply:
I guess it was a merge error